content/blog/2015-04-23-prometheus-posts-around-the-web.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Prometheus posts around the web"
+created_at: 2015-04-23
+kind: article
+author: Brian Brazil
+---
+
+It has been almost 3 months since Prometheus version [0.10.0 was publicly
+announced](https://developers.soundcloud.com/blog/prometheus-monitoring-at-soundcloud).
```

Comment:
I think if we're going to mention version 0.10.0 at all (maybe we shouldn't, and just say that Prometheus was publicy announced?), then it'd also be good to mention that we're now already at 0.13.1.